chat-widget/src/components/livechatwidget/common/reconnectChatHelper.ts

```diff
@@ -47,6 +47,10 @@ const handleChatReconnect = async (chatSDK: any, props: any, dispatch: Dispatch<
         dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.ReconnectChat });
         return;
     }
+
+    // If we have reached this point, it means there is no valid reconnect id or redirectUrl
+    // This is a unauth reconnect refresh scenario - returns true so that we can start normal hydration process
+    return true;
```

Comment:
I have conflicts with returning true for this point, while we return undefined for others, 

is not clear exactly what is the meaning of true, since the purpose of the function is to handle the reconnect, not to validate, and this "true" doesn't really mean the reconnect happened successfully.

could we considerate using ENUMS to pass the scenario reached?

by example, HandleReconnect.NovalidRecconnectIdScenario

Reply:
Using Enum for a helper function with boolean return value is over-engineering IMO. I've changed return type to boolean only and added comment for the return value's meaning